### ISSUE TITLE
deploy branch on Heroku

### ISSUE THREAD

**Issue by thekaysev** [barrycumbie/dev-challenge-01#146]
hey @barrycumbie this is my heroku name: thekaysev

**Comment by barrycumbie** [comment]
I mean, technically your heroku username is your full email address...but you are set up to proceed.

**Comment by thekaysev** [comment]
@Cody-Daffinson

**Comment by barrycumbie** [comment]
> designbydaff@gmail.com

got it , proceed .

**Comment by Cody-Daffinson** [comment]
My active test site: https://designbydaff.herokuapp.com/

**Comment by thekaysev** [comment]
My active link for my dev profile thru Heroku: http://thekaysev.herokuapp.com/docs/dev-profile/sevigne-dev-profile.html

*Cody will be able to submit his when he gets Dev-01 finished and merged into our product

**Comment by thekaysev** [comment]
@barrycumbie Cody and I are working on this and we've gotten everything working until we get to the index.html page thru Heroku. When we get there on both mine and Cody's active link, his Dev-Profile is not showing up on the index page. He was just merged into the REPO today, so from what I can see, it looks like the ```main``` branch is not being updated on Heroku from GitHub. I updated our branch (see [iss146](https://github.com/barrycumbie/dev-challenge-01/tree/iss146)) to be equal with the ```main``` branch on GitHub. Still, nothing is updating. We are running the main branch for Automatic deploys and our branch --iss146-- on Manual deployment. We tried troubleshooting using Heroku CLI commands, which didn't work after we noticed that that isn't the deployment method. I get nervous to do <i>TOO</i> much troubleshooting, because I don't want to accidentally delete Cody's ability to do the assignment at all.
